lib/hexpm/web/views/layout_view.ex

```diff
@@ -42,9 +42,17 @@ defmodule Hexpm.Web.LayoutView do
       tag(:meta, property: "og:title", content: Map.get(assigns, :title)),
       tag(:meta, property: "og:type", content: "website"),
       tag(:meta, property: "og:url", content: Map.get(assigns, :canonical_url)),
-      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/hex.png")),
+      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/open-graph.png")),
+      tag(:meta, property: "og:image:width", content: "1200"),
+      tag(:meta, property: "og:image:height", content: "630"),
       tag(:meta, property: "og:description", content: description(assigns)),
-      tag(:meta, property: "og:site_name", content: "Hex")
+      tag(:meta, property: "og:site_name", content: "Hex"),
+      tag(:meta, property: "twitter:card", content: "summary_large_image"),
+      tag(:meta, property: "twitter:site", content: "@hex_pm"),
+      tag(:meta, property: "twitter:creator", content: "@hex_pm"),
```

Comment:
Yes, I would stick just with `og:*` links since we are not using the full resource from twitter cards. I could also remove the `summary_large_image` type. Since that doesn't make sense for that case.